docs/user-level.md

```diff
@@ -124,7 +127,7 @@ It can be controlled from other graphical apps or from command line using `lncli
 #### Usage
 
 * **Installing `lnd` automatically installs `lnd-unlocker`, which creates a wallet right after installation and stores the seed in `/var/lib/lnd-system-mainnet/.seed.txt`**
-* The above behavior can be overriden by `--no-install-recommends`, but **do not do it unless you prefer false sense of security. Without unlocker `lnd` will stay down after crash. If long enough others may steal your sats!**
+* The above behaviour can be overwritten by `--no-install-recommends`, but **do not do it unless you prefer false sense of security. Without unlocker `lnd` will stay down after crash. If long enough others may steal your sats!**
```

Comment:
I use US English and it'd make sense to keep it consistent across repository.
I believe "overridden" is more appropriate than "overwritten" when referring to behavior.

Reply:
Ok, I'm more used to UK spelling. I will push a spell check with us dictionary.